src/main/resources/config/liquibase/changelog/20230908050001_changelog.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="zbil" id="20230908050001">
+        <createTable tableName="course_code_of_conduct_agreement">
```

Comment:
I think there are multiple foreign keys missing here, however I am not a liquibase notation expert (have you maybe defined them somewhere else?).
1. On Delete Cascade Foreign Key From course_id to table courses.
2. On Delete Cascade Foreign Key from user_id to table users.

Reply:
The new migration adds a foreign key constraint